deep.json

```
@@ -1,7 +1,7 @@
 {
   "package": {
     "name": "@deep-foundation/capacitor-device",
-    "version": "3.0.0"
+    "version": "3.0.1"
```

```suggestion
    "version": "4.0.0"
```

There are breaking changes https://capacitorjs.com/docs/updating/5-0#device